gemini/function-calling/forced_function_calling.ipynb

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if response.candidates[0].content.parts[0].function_call.name == "search_arxiv":
</pre><p>I think you can use <code>response.function_call</code> or something similar to access this easier. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/598/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
I poked around in the SDK and I cannot find that attribute in the function call response. I recall seeing `response.function_name` and `response.function_args` as part of a proposal, but not sure if they've been implemented or shipped yet.